scripts/kanzus_dev

```diff
@@ -0,0 +1,219 @@
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+def start_stills_flow(event):
+    ###RunFlow
+    label = 'SSX_Stills_{}_{}'.format(event['chip_name'],event['cbf_num'])
+    flow = stills_flow.run_flow(flow_input=flow_input, label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+
+def start_publish_flow(event):
+    ###RunFlow
+    label = f'SSX_Plot_{}_{}'.format(event['chip_name'],event['cbf_num'])
+    flow = publish_flow.run_flow(flow_input=base_input,label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+def start_prime_flow(event):                                   
+    ###RunFlow
+    label = 'SSX_Prime_{}_{}'.format(event['chip_name'],event['cbf_num'])                                                                                                                                    
+    flow = prime_flow.run_flow(flow_input=base_input, label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+# Arg Parsing
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('localdir', type=str, default='.')
+    parser.add_argument('--datadir', type=str, 
+        default='/eagle/APSDataAnalysis/SSX/random_start')
+    parser.add_argument('--deployment','-d', default='raf-prod', help=f'Deployment configs. Available: {list(deployment_map.keys())}')
+    return parser.parse_args()
+
+##Gather deployment specific variables
+from gladier_kanzus.deployments import deployment_map
+
+##Import relevant flows
+from gladier_kanzus.flows import TransferFlow
+from gladier_kanzus.flows import StillsFlow
+from gladier_kanzus.flows import PublishFlow
+from gladier_kanzus.flows import PrimeFlow
+
+
+
+if __name__ == '__main__':
+
+    args = parse_args()
+
+    local_dir = args.localdir
+    data_dir = args.datadir
+
+    n_initial_transfer = 512
+    n_batch_transfer = 2048
+    n_batch_stills = 512
+    n_batch_publish =  2048
+    n_batch_prime =  2000
+
+    depl = deployment_map.get(args.deployment)
+    if not depl:
+        raise ValueError(f'Invalid Deployment, deployments available: {list(deployment_map.keys())}')
+
+    depl_input = depl.get_input()
+
+    base_input = {
+        "input": {
+            #Processing variables
+            "base_local_dir": local_dir,
+            "base_data_dir": data_dir,
+            # funcX endpoints
+            'funcx_endpoint_non_compute': depl_input['input']['funcx_endpoint_non_compute'],
+            'funcx_endpoint_compute': depl_input['input']['funcx_endpoint_compute'],
+            # globus endpoints
+            "globus_local_ep": depl_input['input']['beamline_globus_ep'],
+            "globus_dest_ep": depl_input['input']['theta_globus_ep'], 
+    	    "globus_dest_mount" : depl_input['input']['ssx_eagle_mount'],
+        }
+    }
```

Comment:
These values could all use some docs. I think `base_local_dir` might be where the file watcher looks, but I can't tell what `base_data_dir` does. 

The amount of vars required is nice though -- it's great that there are only 2 args that need to change, and the rest are deployment.